pom.xml

```diff
@@ -39,69 +39,30 @@
     </scm>
 
     <repositories>
-        
         <repository>
             <id>atlas-nexus-01-totalfreedom-development</id>
             <url>https://nexus-01.core.atlas-media.co.uk/repository/totalfreedom-development/</url>
         </repository>
 
-        <repository>
-            <id>CodeMC</id>
-            <url>https://repo.codemc.org/repository/maven-public/</url>
-        </repository>
-
-        <repository>
-            <id>spigot-repo</id>
-            <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
-        </repository>
-
         <repository>
             <id>enginehub</id>
             <url>https://maven.enginehub.org/repo/</url>
         </repository>
 
-        <repository>
-            <id>elmakers-repo</id>
-            <url>https://maven.elmakers.com/repository/</url>
-        </repository>
-
-        <repository>
-            <id>sk89q-snapshots</id>
-            <url>https://maven.sk89q.com/artifactory/repo</url>
-        </repository>
-
-        <repository>
-            <id>dv8tion</id>
-            <name>m2-dv8tion</name>
-            <url>https://m2.dv8tion.net/releases/</url>
-        </repository>
-
         <repository>
             <id>playpro</id>
             <url>https://maven.playpro.com/</url>
         </repository>
 
         <repository>
-            <id>md_5-public</id>
-            <url>https://repo.md-5.net/content/groups/public/</url>
-        </repository>
-
-        <repository>
-            <id>dmulloy2-repo</id>
-            <url>https://repo.dmulloy2.net/nexus/repository/public/</url>
-        </repository>
-
-        <repository>
-            <id>sk89q-repo</id>
-            <url>https://maven.sk89q.com/repo/</url>
+            <id>essentialsx-repo</id>
+            <url>https://repo.essentialsx.net/releases/</url>
         </repository>
 
         <repository>
-            <id>esentialsx-repo</id>
```

Comment:
Generally speaking we have tried to remove Jitpack due to it's inability to operate as a stable / reliable service combined with some security issues (If I recall correctly) where packages could potentially be tampered with.

Reply:
I 100% agree with you on that one. I think Jitpack has a huge possibility for being a possible vector for a supply chain attack if you don't manage it correctly.

That's why it's been added at the bottom of the repositories, which I hope will only allow it to be used as a fallback when the Atlas Nexus is down, as it would with Gradle. If I recall correctly all of the possible Jitpacked dependencies are all in-house which I think should minimise any possibility of a supply chain attack unless the entirety of Jitpack is compromised.